crates/hyperswitch_connectors/src/connectors/facilitapay/transformers.rs

```diff
@@ -372,9 +374,14 @@ impl<F, T> TryFrom<ResponseRouterData<F, FacilitapayCustomerResponse, T, Payment
         item: ResponseRouterData<F, FacilitapayCustomerResponse, T, PaymentsResponseData>,
     ) -> Result<Self, Self::Error> {
         Ok(Self {
-            response: Ok(PaymentsResponseData::ConnectorCustomerResponse {
-                connector_customer_id: item.response.data.customer_id.expose(),
-            }),
+            response: Ok(PaymentsResponseData::ConnectorCustomerResponse(
+                ConnectorCustomerResponseData {
```

Comment:
This way, any new field addition to `ConnectorCustomerResponseData` will not lead to changes in these connector code.

```suggestion
                ConnectorCustomerResponseData::new_with_customer_id(item.response.data.customer_id.expose())
```